robust_pdf_plot.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #141.        if FIG_DIR and LATEXIFY and len(save_name) &gt; 0:
</pre><p>Again <code>FIG_DIR</code> is not needed here, because <code>FIG_DIR</code> check is inbuilt in <code>savefig</code> command.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Removed FIG_DIR check, only LATEXIFY check

Reply:
Fixed